src/bespokelabs/curator/prompter/prompter.py

```diff
@@ -113,13 +113,14 @@ def _completions(
         )
 
         prompt_func_hash = _get_function_hash(self.prompt_formatter.prompt_func)
+
+        # Used as the name of the dataset .arrow file so changing this will keep the same cache for the LLM responses, but will create a new dataset file
```

Comment:
Line too long. Please split.
(Google convention is 80 characters per line for Python).